packages/libz/libz.spec

```diff
@@ -0,0 +1,60 @@
+Name: %{_cross_os}libz
+Version: 1.2.11
+Release: 1%{?dist}
+Summary: Compression and decompression library
+URL: https://www.zlib.net/
+License: zlib and Boost
+
+Source: http://www.zlib.net/zlib-%{version}.tar.gz
+
+BuildRequires: %{_cross_os}glibc-devel
+
+%description
+%{summary}.
+
+%package devel
+Summary: Header files and libraries for Zlib development
+Requires: %{name}
+
+%description devel
+%{summary}.
+
+%prep
+%setup -n zlib-%{version}
+
+# Sets cross build flags, target cross compiler, and env variables
+# required to `make install` libz
+%global set_env \
+%set_cross_build_flags \\\
+export CC="%{_cross_target}-gcc" \\\
+export DESTDIR='%{buildroot}' \\\
+export PREFIX='%{_cross_prefix}' \\\
```

Comment:
I don't see this being used in the zlib build, just the `--prefix` argument you have below - is this needed?  We could pass `%{_cross_prefix}` below.